sql/catalyst/src/test/scala/org/apache/spark/sql/connector/catalog/InMemoryTable.scala

```diff
@@ -35,6 +36,7 @@ class InMemoryTable(
     schema: StructType,
     override val partitioning: Array[Transform],
     override val properties: util.Map[String, String],
+    override val constraints: Array[Constraint] = Array.empty,
```

Comment:
Following the comment to add constraints capability, so currently added constraints are ignored in `InMemoryTable`, are you going to implement the feature later?